src/App.tsx

```diff
@@ -1,9 +1,11 @@
 import { PokemonsList } from './PokemonsList';
+import { HeaderText } from './HeaderText';
 import './App.css';
 
 export function App(): JSX.Element {
   return (
     <div className="App">
+      <HeaderText />
```

Comment:
It is better to inline `<h1 className="header-text">Choose two pokemons</h1>` here. No need in creating a component just for one line.